enzy_htp/_config/pymol_config.py

```diff
@@ -0,0 +1,51 @@
+"""Defines PyMOLConfig() which holds configuration settings for enzy_htp to interface with the 
```

Comment:
I also made a pymol interface. This will cause conflict when I merge it. Maybe we can merge our code manually first? Or any better ways to do this

Reply:
I will add diffs in my branch.